src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/ThirdPartyUser.razor

```diff
@@ -56,11 +56,11 @@
                     case (nameof(ThirdPartyUserDto.IdpDetailDto)):
                         if (context.Item.IdpDetailDto.Name == LdapConsts.LDAP_NAME)
                         {
-                            <MIcon Size="40">@context.Item.IdpDetailDto.Icon</MIcon>
+                            <MIcon Size="24" Style="color: #323D6F !important;">@context.Item.IdpDetailDto.Icon</MIcon>
```

Comment:
#323D6F  not recommended,replace class in stack component

Reply:
ok